src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      name="form-search--big"
```

Comment:
Follow the checklist
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/62e5a7f9-e122-45d6-ad9f-0b9b62118858)